test/e2e_tests/e2e_test.go

```diff
@@ -137,4 +144,28 @@ var _ = Describe("validate NFSPVC controller functionality", func() {
 		By("deleting the NFSPVC")
 		utilst.DeleteNfsPvc(k8sClient, desiredNfsPvc)
 	})
+
+	It("Should add the nfs version to the mountOption in the pv", func() {
```

Comment:
add a test to make sure the mountoptions are not added if no nfs version is given